frontend/src/components/Statistics/SubmissionPieChart.js

```diff
@@ -34,7 +29,9 @@ const SubmissionPieChart = () => {
   useEffect(() => {
     const fetchDataAndInitializePieChart = async () => {
       try {
-        const historyResponse = await getPieChartData(await getCookie(), await getUserId());
+        const jwt = getCookie();
+        const userId = await getUserId();
+        const historyResponse = await getPieChartData(jwt, userId);
```

Comment:
`getCookie()` is no longer awaited. Would this be a problem?

Reply:
getCookie dont require await